### PR TITLE
Pre trained keras

### PR DESCRIPTION
**Pull request by martham93**
pre-trained commonly used weights for `Keras` and `PyTorch`

### CODE REVIEW

File: pyveda/pretrainedmodels.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
+"""ResNet50 model for Keras.
```
Comment (chelm) [inline pyveda/pretrainedmodels.py]: So this file goes away right?

**Comment by chelm** [comment]
@martham93 Can you rename the `keras_models` and `pytorch_models` to be just `keras` and `pytorch`? 

So the imports are `from pyveda.pretrainedmodels.keras import ResNet50`

**Comment by chelm** [comment]
Also do you think we a model type in import/paths? Im not sure, but something like:
`from pyveda.pretrainedmodels.keras.classification import ResNet50` and `from pyveda.pretrainedmodels.pytorch.segmentation import UNet`

File: pyveda/pretrainedmodels.py (hunk 1 above)
Comment (martham93) [reply pyveda/pretrainedmodels.py]: yes!

**Pull request closed without merging**